companion/src/modeledit/modeledit.cpp

```diff
@@ -120,9 +120,14 @@ void ModelEdit::on_pushButton_clicked()
 
 void ModelEdit::launchSimulation()
 {
-  RadioData *simuData = new RadioData();
+  RadioData * simuData = new RadioData();
   simuData->generalSettings = generalSettings;
-  simuData->models[0] = model;
+  if (simuData->models.size() == 0) {
+    simuData->models.push_back(model);
+  }
+  else {
+    simuData->models[0] = model;
+  }
```

Comment:
This is the single model simulator launched from the model editor. Yes it was always like that there.

The button on the model list window does the whole radio, the button in a model does just that model.